docs/devtool/git.md

```diff
@@ -1,3 +1,97 @@
 (label:git)=
 
 # Git
+Git allows developers to keep track of changes to their code, collaborate with other developers, and work on multiple versions of their software simultaneously. This is achieved through a distributed model, where each developer has a copy of the codebase on their local machine and can make changes and commit them to a shared repository.
+
+## Install Git
+You can download Git from the official website: https://git-scm.com/downloads
+
+If you use Mac, first install [Homebrew](https://brew.sh/index_zh-tw.html), you can use Homebrew to install Git, and execute this line of command in the terminal to complete:
+```
+$ brew install git
+```
+
+## Set up the Git environment
+Once Git is installed, you'll need to set up your environment by configuring your username and email address, so that Git can attribute your changes to you. To do this, open a command prompt or terminal window and type the following commands, replacing "Your Name" and "Youremail" with your own information:
+```
+$ git config --global user.name "Your Name"
+$ git config --global user.email "Youremail@example.com"
+```
+
+## Git workflow and basic commands
+<img width="820" alt="image" src="https://user-images.githubusercontent.com/123142931/219302278-e9655f60-d6b6-4594-a0fc-4a649c6e11d0.png">
```

Comment:
Can you please place the image in the `group_manual/docs/image/` directory, and using the figure syntax as in [this file](https://raw.githubusercontent.com/wengroup/group_manual/main/docs/contributing/edit.md) to refer to it?

Reply:
Done